docs/guidelines/platforms/android/guideline_percievable_android.md

```diff
@@ -164,41 +252,28 @@ Grouping elements based on the context helps users that benefit from using acces
 </ConstraintLayout>
 ```
 
-_Note:_ In cases like this, it is recommended to define content descriptions as concisely as possible, considering that accessibility services will read them one after the other.
-
-- Custom group label
-
-It is possible to override the platform's default grouping and ordering of a group's inner element descriptions by providing a content description for the group itself.
-
-**Code example:**
-
+In Compose, the same can be achieved using the `mergeDescendants` parameter:
+```kotlin
+@Composable
+private fun SongCard(title: String, artist: String) {
+    Column(modifier = Modifier.semantics(mergeDescendants = true) {}) {
+        Text(title)
+        Text(artist)
+    }
+}
 ```
-<!-- In response to a single user interaction, accessibility services announce the custom content description for the group. -->
```

Comment:
I removed this example, since I couldn't think of a usecase for this other than combining the descriptions of child elements, which is discouraged by the [docs](https://developer.android.com/guide/topics/ui/accessibility/principles#content-groups).

Left it as a smaller comment/note, just in case someone actually needs it.